lib/pavilion/plugins/system/sys_arch.py

```diff
@@ -0,0 +1,20 @@
+import subprocess
+import pavilion.system_plugins as system_plugins
+
+class SystemArch( system_plugins.SystemPlugins ):
+
+    def __init__( self ):
+        super.__init__( 'sys_arch', 10, False, [ None ] )
```

Comment:
While I'm complaining, you should generally pass kwargs with as `name=value`. It just helps readability.